abstract_commands.xml

```diff
@@ -2,13 +2,13 @@
     <register name="Access Register">
         \begin{steps}{Perform the following sequence of operations:}
         \item Halt the hart, if \Fprehalt is set. If the hart is already
-            handled, the entire command fails.
+            halted, the entire command fails.
         \item Execute the Program Buffer, if \Fpreexec is set.
         \item Copy data from the register specified by \Fregno into {\tt data}, if \Fwrite is clear.
         \item Copy data from {\tt data} into the register specified by \Fregno, if \Fwrite is set.
         \item Execute the Program Buffer, if \Fpostexec is set.
-        \item Resume the hart, if \Fpostresume is set. If the hart is already
-            running, the entire command fails.
+        \item Resume the hart, if \Fpostresume is set. If the hart is already 
+        running and \Fprehalt is not set, the entire command fails.
```

Comment:
Why do you need the failure be conditional on \Fprehalt?

I imagined you could do a write-register-and-resume command, but if you do that while the hart's already running it should fail when it discovers that. I realize the phrasing is ambiguous. What I meant:
```If \Fpostresume is set: fail the command if the hart is already running, otherwise resume the hart.```

Does that make sense?